trainTestReinforcementAlgorithm.py

```diff
@@ -115,15 +138,39 @@ def saveRL(
     model.save(model_filename)  
 
 
+# %% [markdown]
+# ## Initialize Game Variables
+# %%
+# Set amount of time for training/testing. One step is one action for the snake.
+train_timesteps=1000
+test_timesteps=100
+
+# Set gym environment name
+env_name = 'snake-v0'
+
+# Set board dimensions
+board_height = 10
+board_width = 10
+
+# Set visualization arguments
+visualize_training = False # We don't want to visualize the training process
+visualize_testing = True # Set to true in order to see game moves in pygame. Should be false if run on server.
+visualization_fps = 30
+
+# Set reward function to be used in training 
+# Reward functions are defined in snakeRewardFuncs.py
+reward_function = basic_reward_func 
+
+
 # %% [markdown]
 # ## Run in Notebook
 # To run in the notebook, uncomment the following three lines:
 
 # %%
-# model = trainRL()
-# scores = testRL(model)
-# analyzeRL(scores)
-# saveRL(model)
+model = trainRL(train_timesteps, env_name, board_height, board_width, visualize_training, visualization_fps, reward_function)
```

Comment:
Use defaults (sorry for repetition):
```suggestion
model = trainRL()
```